lib/ansible/modules/fetch.py

```diff
@@ -56,6 +56,18 @@
     - If using multiple hosts with the same filename, the file will be overwritten for each host.
     type: bool
     default: no
+  encrypt:
```

Comment:
not sure we need 2 values here, I would add `vault:` parameter, it should either match a vault id OR be the vault secret itself. 

When not specified, no encryption is used.

Reply:
Makes sense, will change.

Reply:
One downside of this is that if the user makes a typo in vault id, then there is no error, the file is silently encrypted with that string as a secret, and the "secret" is then plainly visible in the playbook. Possibly insecure behavior. What about two params, vault_id and secret, and if any of the two is defined, encryption is enabled? (Does it make sense to have both defined? Can we specify secret and vault_id independently? Won't be there problems with decryption then?)